roles/edpm_network_config/tasks/os_net_config.yml

```diff
@@ -30,6 +29,13 @@
         dest: "{{ nic_config_file }}"
         mode: '0644'
         backup: true
+    - name: Debug display contents of nic_config_file before applying
+      ansible.builtin.command: "cat {{ nic_config_file }}"
+      register: os_net_config_config
+      changed_when: false
+    - name: Print to console
+      ansible.builtin.debug:
+        msg: "{{ os_net_config_config.stdout }}"
```

Comment:
Looks good to me. I have two suggestions:

* Instead of cat. Can we switch to slurp module https://docs.ansible.com/ansible/latest/collections/ansible/builtin/slurp_module.html instead of cat command?
* Can we add a var so that end user can enable it when they debug output?

Reply:
Another +1 to both points.
@marios I think that if you fix these two points, this PR will be an easy merge.

Reply:
ack sure will update